test/testinfra.py

```diff
@@ -401,22 +423,81 @@ def update_status(revision, context, last, changes):
                     if status.get("description", NO_TESTING) == NO_TESTING:
                         baseline = 0
 
-                (priority, changes) = self.prioritize(status, labels, baseline)
+                (priority, changes) = self.prioritize(status, labels, baseline, context)
                 if update_status(revision, context, status, changes):
-                    results.append((priority, "pull-%d" % number, revision, "pull/%d/head" % number, context))
+                    results.append(GitHub.TaskEntry(priority, GithubPullTask("pull-%d" % number, revision,
+                                                                             "pull/%d/head" % number, context)))
+
+    def scan_for_image_tasks(self):
+        # XXX
+        return [ (10, GithubImageTask("refresh-fedora-23", "fedora-23")) ]
+
+        issues = self.get("issues?labels=bot")
+
+        results = [ ]
+        for image in DEFAULT_IMAGE_REFRESH:
```

Comment:
This doesn't take into consideration that an image might be set to False. If it's used like this, it should be an array or list.